src/custom/GoogleDLPRedactor.ts

```diff
@@ -121,13 +120,24 @@ export interface GoogleDLPRedactorOptions {
 
 /** @public */
 export class GoogleDLPRedactor implements IAsyncRedactor {
-  dlpClient: typeof DLP.DlpServiceClient;
+  dlpClient: any;
+  dlpClientOpts: any;
 
   constructor(private opts: GoogleDLPRedactorOptions = {}) {
+    this.dlpClientOpts = opts;
+    this.dlpClient = {};
+  }
+
+  async setup(): Promise<any> {
+    const DLP = await import('@google-cloud/dlp');
```

Comment:
hey, I see what you're trying to do, but honestly I don't like that approach. The problem is this is kind of lazy initializing the GoogleDLPRedactor redactor which means an app that uses redact-pii with DLP redaction "might" look like its working at first but only fails later down the road when it's redacting some data. 
It'd rather like to fail an app on bootstrap if one forgets to install the peerDependency when using GoogleDLPRedactor.

Reply:
I had myself some painful experiences months ago with a mongodb library that was lazily initializing some stuff and made it unnecessarily hard to catch cases where it could not connect to mongodb.

Reply:
I think the best is really to remove the GoogleDLP redactor from the main module entrypoint and update the docs to mention that the GoogleDLPRedactor has to be imported via `from `redact-pii/lib/custom/GoogleDLPRedactor`. Eventually we should probably put the DLP redactor into it's own package, but I was too lazy to do that for now because of the additional dev plumbing overhead.

Reply:
This is excellent feedback. Can you provide a recommended solution in broad terms and I can give it a shot? Would it be safe to import the package as long as only types are used?